README.md

```diff
@@ -2,4 +2,21 @@
 
 # SUPERDELEGATION
 
-## NEEDS A README
+## Getting Started
+0. Clone the repo and cd into it
+1. Install [rbenv](https://github.com/rbenv/rbenv#installation)
```

Comment:
I think officially we've been using RVM? Or does it not matter? If not, maybe mention that either is fine?